### PR TITLE
Improve package

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 
 
+## 0.3.0 - 2016-07-18
+
+
+## 0.2.1 - 2016-07-18
+
+### Changed
+
+- Updated discovery dependency
```
Comment (Nyholm) [inline CHANGELOG.md]: Sorry, I forgot about changelog again...

Comment (sagikazarmark) [inline CHANGELOG.md]: np  :smile:

**Comment by Nyholm** [comment]
I see that you've prepared to release 0.3.0. I see no need for that. Not until there is a code change.

**Comment by sagikazarmark** [comment]
Why is travis not working?

**Comment by Nyholm** [comment]
It is slow.. It has been for the last 4 hours.. Packagist is also having issues...

**Comment by dbu** [comment]
> It is slow.. It has been for the last 4 hours.. Packagist is also having
> issues...

with the amounts of merges and tagging, it might also be that we have a
very full pipeline. afaik each organization has a limited number of
parallel builds, so when a lot happens, things tend to get clogged up.
it should catch up over the night or whenever we stop doing so many
chances :-)

**Comment by Nyholm** [comment]
That is true, but I've experienced slow travis on other organisations today as well..

File: README.md [hunk 2]
```diff
@@ -18,38 +18,11 @@ Via Composer
...
 ```
 
-## Features
 
-This lib adapt ReactPHP behaviour to use the [PSR7 interfaces](https://github.com/php-fig/http-message).
-Also `sync` and `async` requests are possible without more code than a function call.
+## Documentation
 
-For a deeper `async` comprehension, you must check how ReactPHP engine work.
+Please see the [official documentation](http://docs.php-http.org/en/latest/clients/guzzle5-adapter.html).
```
Comment (Nyholm) [inline README.md]: http://docs.php-http.org/en/latest/clients/react-adapter.html

**Comment by Nyholm** [comment]
Good, just update the link and I'm happy to merge

File: README.md (hunk 2 above)
Comment (sagikazarmark) [inline README.md]: Ah, I concentrated so hard no to mess it up....

**Pull request merged**